settings.py

```diff
@@ -0,0 +1,14 @@
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "FastAPI city-temperature manager"
+
+    DATABASE_URL: str | None = "sqlite:///./city_temperature.db"
```

Comment:
The type annotation `str | None` is not compatible with Python versions below 3.10. Consider using `Optional[str]` if you are using a version below 3.10.